ote/src/clj/ote/services/transport.clj

```diff
@@ -471,24 +483,51 @@
       (log/info (str "Delete association failed with service: " service-id " and operator: " operator-id))
       {:status 404 :body "Association not found"})))
 
+(defn public-data-transport-service
+  "Get single transport service by id"
+  [db id]
+  (let [ts (first (fetch db ::t-service/transport-service
+                         (apply disj
```

Comment:
I'm wondering what are the benefits between 
a) just defining the allowed columns
vs. 
b) taking "all" columns and disjoinin the "not-allowed" ones like done here.

In the first case it's a bit more quaranteed that no unwanted column gets to the result set accidentally because one has to add them explicitly, 
in this case any new column gets included by default.

Which ever design path should be taken? If b) is taken, then this solution looks good.

Reply:
I used this approach because I tought it was just temporary that we couldn't return everything.